mllib/src/main/scala/org/apache/spark/mllib/api/python/PythonMLLibAPI.scala

```diff
@@ -465,6 +465,36 @@ private[python] class PythonMLLibAPI extends Serializable {
   }
 
   /**
+   * Java stub for Python mllib LDA.run()
+   */
+  def trainLDAModel(
+      data: JavaRDD[LabeledPoint],
```

Comment:
I haven't tested mine with tuples yet; you're right we should try that.

Mine should support arrays, Vectors (dense & sparse), numpy, and scipy types since it passes everything through _convert_to_vector.